docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -170,7 +170,7 @@ Add the following line to /etc/apt/sources.list:
 
 .. code-block:: bash
 
-    deb http://ppa.launchpad.net/ansible/ansible/ubuntu trusty main
+    $ deb http://ppa.launchpad.net/ansible/ansible/ubuntu trusty main
```

Comment:
this is a string in a file, not a bash command.

Reply:
my bad.